src/Microsoft.Identity.Client/Platforms/Android/AndroidWebUIFactory.cs

```diff
@@ -4,14 +4,15 @@
 using Microsoft.Identity.Client.Core;
 using Microsoft.Identity.Client.Platforms.Android.EmbeddedWebview;
 using Microsoft.Identity.Client.Platforms.Android.SystemWebview;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
 using Microsoft.Identity.Client.UI;
 
 namespace Microsoft.Identity.Client.Platforms.Android
 {
     [global::Android.Runtime.Preserve(AllMembers = true)]
     internal class AndroidWebUIFactory : IWebUIFactory
     {
-        public IWebUI CreateAuthenticationDialog(CoreUIParent coreUIParent, RequestContext requestContext)
+        public IWebUI CreateAuthenticationDialog(CoreUIParent coreUIParent, RequestContext requestContext, IPlatformProxy platformProxy)
```

Comment:
>RequestContext requestContext, IPlatformProxy platformProxy [](start = 76, length = 59)

See suggestion elsewhere, can the proxy or perhaps even the servicebundle be referenced in the requestContext instead of adding another parameter?